### DISCUSSION TITLE
Alpha support in color selector

### DISCUSSION THREAD

**Discussion by mrhasbean** [pods-framework/pods#6781] [Ideas]
With recent trends and updates to page builders, alpha selectors are being added to most colour fields, but we presently have no method in Pods to support that functionality.

The ideal solution would be to add the alpha selector to the colour field but as an option that can be enabled when the field is defined (checkbox in Additional Field Options). Field data would have to be stored appropriately determined by whether the alpha channel is enabled or not.

**Comment by quasel** [comment]
Valid Point!
